TestApi/TestApi.DAL/Commands/AllocationService.cs

```diff
@@ -119,28 +131,21 @@ private async Task<Allocation> GetAllocationByUserId(Guid userId)
 
         private async Task CreateNewAllocation(Guid userId)
         {
-            var createNewAllocationCommand = new CreateNewAllocationByUserIdCommand(userId); 
-            await _commandHandler.Handle(createNewAllocationCommand);
+            var command = new CreateNewAllocationByUserIdCommand(userId); 
+            await _commandHandler.Handle(command);
         }
 
-        private async Task<User> GetUnallocatedUser(IEnumerable<User> users)
+        private static Guid GetUnallocatedUser(IEnumerable<Allocation> allocations)
```

Comment:
A nullable guid response or out parameter would be better here than to check for an empty Guid

Reply:
aha! I forgot you can do Guid? for nullable 👍 Changing...